tests/nixos/default.nix

```
@@ -36,7 +36,7 @@ let
           # NOTE: noTests pkg might not have been built yet for some older versions of the package
           #       and in versions before 2.25, the untested build wasn't shared with the tested build yet
           #       Add noTests here when those versions become irrelevant.
-          nix = (builtins.getFlake "nix/${nixVersion}").packages.${system}.nix;
+          nix = (builtins.getFlake "nix/${nixVersion}").packages.${system}.nix-cli;
```

Older versions don't have `nix-cli` yet.

```suggestion
          nix = let
            packages = builtins.getFlake "nix/${nixVersion}").packages.${system};
          in packages.nix-cli or packages.nix;
```